src/app/space/analyze/analyze-overview/analyze-overview.component.ts

```diff
@@ -15,7 +15,7 @@ import { AuthenticationService, User, UserService } from 'ngx-login-client';
 })
 export class AnalyzeOverviewComponent implements OnInit, OnDestroy {
   context: Context;
-  isUserEnabled: boolean = false;
+  _isUserEnabled: boolean;
```

Comment:
Does leaving this `undefined` have any functional effect on the `*ngIf`? I'd expect it to still evaluate falsy, same as if it were initialized to `false`.

Reply:
I altered the `*ngIf` to use a function that was added as well. Or at least I should have; I seem to be making random mistakes with merging commits and what not today.

Reply:
Right, but as far as the `*ngIf` conditional goes, `false`, `null`, and `undefined` all evaluate falsy and produce the same result. So the expression used in the new function is going to produce the same result in the `undefined` case as the old case of just initializing to `false` off the bat.

Reply:
Oh... yeah good point.

Reply:
Right, they need to actually be different for the two divs. Bleh.

Reply:
Bleh; I was looking for a small, neat fix to cover the three states.